app/home/home.js

```diff
@@ -1,14 +1,13 @@
-viewsModule.config(function($routeProvider) {
+viewsModule.config(['$routeProvider', function($routeProvider) {
   $routeProvider.when("/", {
     templateUrl : "./home/home.html",
     controller : 'HomeCtrl'
   });
-});
+}]);
 
-viewsModule.controller('HomeCtrl', function($scope, owmUSCities, owmHistory) {
-  owmHistory.push({ name : "Home", isHome : true });
+viewsModule.controller('HomeCtrl', ['$scope', 'owmUSCities', function($scope, owmUSCities) {
```

Comment:
absolutely agreed but in this particular task we are teaching the student about annotating the dependencies that is why they are done manual. But I agree there should be a part in which we should introduce [ng-annotate](https://github.com/olov/ng-annotate) via Grunt or Gulp so that the student can learn about the automation options.